src/accelerate/test_utils/scripts/external_deps/test_metrics.py

```diff
@@ -255,6 +269,9 @@ def main():
     accelerator = Accelerator()
     test_torch_metrics(accelerator, 512)
     accelerator.state._reset_state()
+    if accelerator.is_local_main_process:
+        print("**Test that `drop_last` is taken into account**")
```

Comment:
Could use `accelerator.print` instead of `if ...`, right? :)

Reply:
We *can*, I can do all of those in a follow-up PR 😉

Reply:
Really not necessary, was more in jest :-p